MaxText/maxengine.py

```diff
@@ -257,6 +263,13 @@ def apply_adapter(self, base_params, adapter_config, adapter_params):
     lora_scale_factor = float(adapter_config["lora_alpha"]) / lora_rank
     lora_utils.apply_lora_on_base_params(base_params, adapter_params, lora_scale_factor)
 
+  def unapply_adapter(self, base_params, adapter_config, adapter_params):
```

Comment:
Is this supposed to be called somewhere?

Reply:
Yes. It will be called from the JetStream Engine.